keylime.conf

```diff
@@ -40,6 +40,9 @@ registrar_port = 8890
 # If rsa_keyname is not a absolute path it is relative to /var/lib/keylime/secure.
 rsa_keyname = tci_rsa_key
 
+# Enable mTLS communication between agent, verifier and tenant.
+mtls_cert_enabled = True
```

Comment:
Add a reference why disabling this is generally a bad idea (https://github.com/keylime/keylime/security/advisories/GHSA-2m39-75g9-ff5r).

Reply:
done

Reply:
The wording is a bit confusing. It makes it seem like setting `mtls_cert_enabled` to `True` is considered insecure. Maybe it should say something like "Details on why setting this to False is considered insecure"... or similar.